parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageWriteStore.java

```diff
@@ -241,6 +265,30 @@ public void writePageV2(
         BytesInput data,
         Statistics<?> statistics)
         throws IOException {
+      writePageV2(
+          rowCount,
+          nullCount,
+          valueCount,
+          repetitionLevels,
+          definitionLevels,
+          dataEncoding,
+          data,
+          statistics,
+          null);
```

Comment:
not sure of coding conventions in parquet-mr but I find parameter comments (`/*size_statistics=*/null`)  can help readability.